tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1648,5 +1648,75 @@ describe("Service Form Modal", function() {
         });
       });
     });
+
+    context("Multi-container (pod)", function() {
+      beforeEach(function() {
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .click()
+          .get(".menu-tabbed-view h2")
+          .contains("Service");
+      });
+
+      it("Should add new container", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+      });
+
+      it("Should contain two containers navigation under Services tab", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
+
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .siblings()
+          .should("to.have.length", 2);
```

Comment:
The suggestion doesn't work.

I also believe that having expect it's more like a helper and not something we should be looking to use all the time because cypress API give us enough for majority of tests.